src/framework/components/system.js

```diff
@@ -21,6 +23,9 @@ function ComponentSystem(app) {
     // The store where all pc.ComponentData objects are kept
     this.store = {};
     this.schema = [];
+
+    // cached mesh primitives
+    this.primitiveMap = new Map();
```

Comment:
This feels like it should be somewhere graphics related and not in the ComponentSystem?

Reply:
done, thanks!